src/SnehalW/AllDifferentIndex.java

```diff
@@ -0,0 +1,28 @@
+package SnehalW;
+
+public class AllDifferentIndex {
+
+	int array1[] = {10,2,9,14,3};
+	int array2[] = {10,2,18,14,3};
+	int i ,j;
+		void differIndex()
```

Comment:
Hi @SnehalWAug19 
Here why we take two for loop in single for loop work can be done..
think you need to compare same index number in both array... for that you should write one for loop and check for which index number are not matching...you are getting my point ?

Please Correct the code!

Reply:
Yes harsh, you r correct program is working by taking  for loop also.Updated the program and pushed in git again